promypy/main.py

```diff
@@ -34,18 +34,23 @@ def write_to_file(file: Optional[str], filenames: Set[str]) -> None:
         echo(output_filenames, file=fh)
 
 
+def call_mypy_api(args: list[str]) -> Tuple[str, Any]:
+    return args[0], api.run([*args])
+
+
 @app.command()
 def dump(
     directory: str = ".",
     mypy_args: Optional[str] = None,
-    timeout: int = 10,
+    timeout: int = 30,
```

Comment:
Why the default changed?

Reply:
mypy was too slow to process even the simplest nested files. Since there were huge number of files which were resulting in timeout exception, we had to change the default instead of passing as an arg.